src/js/server/handlebars.js

```diff
@@ -0,0 +1,77 @@
+// A module that add support for handlebars itself to an express module.
+//
+// This is designed to be used by adding it to a `gpii.express` instance as a child component.
+//
+// Any "helper" functions should extend the `gpii.express.hb.helper` grade, and should be added as child components of an instance of this grade.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.hb");
+
+var exphbs      = require("express-handlebars");
```

Comment:
Yes, my OCD habit of indenting the right-hand side of various assignments falls apart when I take a line or two out.  Fixed.